components/style/gecko_bindings/sugar/ns_t_array.rs

```diff
@@ -93,4 +93,13 @@ impl<T> nsTArray<T> {
         let mut header = self.header_mut();
         header.mLength = len;
     }
+
+    /// Resizes an array containing only POD elements
+    ///
+    /// This is safe since uninitialized C-obtained POD elements are fine
+    pub fn set_len_pod(&mut self, len: u32) {
```

Comment:
Even with `where T: Copy` this is unsafe in Rust for types where not all bit sequences are valid, like`bool`.